c_src/unifex/cnode/unifex/unifex.h

```diff
@@ -40,8 +40,20 @@ typedef struct UnifexCNodeContext {
   UNIFEX_TERM error;
 } UnifexEnv;
 
-void *unifex_alloc(size_t size);
-void unifex_free(void *pointer);
+static inline void *unifex_alloc(size_t size) { return malloc(size); }
+
+static inline void unifex_free(void *pointer) { free(pointer); }
+
+static inline UnifexEnv *unifex_alloc_env(UnifexEnv *env) { return env; }
+
+static inline void unifex_clear_env(UnifexEnv *env) { UNIFEX_UNUSED(env); }
```

Comment:
```suggestion
static inline void unifex_clear_env(UnifexEnv *_env) { UNIFEX_UNUSED(_env); }
```

we use to prefix unused variables with underscore either